rj_gameplay/rj_gameplay/gameplay_node.py

```diff
@@ -88,14 +108,80 @@ def gameplay_tick(self) -> None:
         """
         ticks the gameplay coordinator using recent world_state
         """
-        if self.partial_world_state is not None and self.field is not None:
-
+        if self.partial_world_state is not None and self.field is not None and len(self.robot_statuses) >= NUM_ROBOTS:
             self.world_state = conv.worldstate_creator(self.partial_world_state, self.robot_statuses, self.game_info, self.field)
+        else:
+            self.world_state = None
+
+        #self.get_logger().info("Gameplay node is ticked")
+
+        #self.get_logger().info("Gameplay node is ticked")
 
         if self.world_state is not None:
-            pass
-            # self.gameplay.tick(self.world_state)
+            # self.debug_async_lineup(self.world_state)
+            # self.tick_override_actions(self.world_state)
+            intents = self.gameplay.tick(self.world_state)
+            for i in range(NUM_ROBOTS):
+                self.robot_intent_pubs[i].publish(intents[i])
             # Uncomment when a real play selector is created
+        else:
+            pass
+            #self.get_logger().info("World state was none!")
```

Comment:
Dead code! Uncomment this, remove the pass, and maybe make this a `warn` rather than `info`.